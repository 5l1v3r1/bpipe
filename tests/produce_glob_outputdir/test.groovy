fastqc = {
    output.dir="ABC"
    produce("*.fastqc.zip") {
        exec "touch $output.dir/test.fastqc.zip"
    }
}
run {
    fastqc
}
