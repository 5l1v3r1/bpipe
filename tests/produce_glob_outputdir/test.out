====================================================================================================
|                              Starting Pipeline at 2014-06-21 16:30                               |
====================================================================================================

=========================================== Stage fastqc ===========================================

======================================== Pipeline Finished =========================================
MSG: Generated report /Users/simon/Documents/workspace3/bpipe/tests/produce_glob_jason/doc/index.html
16:30:17 MSG:  Finished at Sat Jun 21 16:30:17 EST 2014
16:30:17 MSG:  Output is ABC/test.fastqc.zip
